src/viam/sdk/rpc/dial.cpp

```diff
@@ -56,6 +59,10 @@ void DialOptions::set_entity(boost::optional<std::string> entity) {
     auth_entity_ = entity;
 }
 
+void DialOptions::set_timeout(std::chrono::seconds timeout) {
```

Comment:
Just as a good practice, either 1) take the parameter as `const&`, or 2) continue to take it as a value but then `std::move` it into `timeout_`. Yes, it doesn't matter for this particular type and use-case much, but it is just a good habit to get into.

Reply:
Ah good point. Chose option 2, as I've seen that `std::move` pattern quite a bit throughout the SDK.